package.json

```diff
@@ -487,6 +487,11 @@
                   "maxStructFields": -1
                 }
               },
+              "dlvToolPath": {
```

Comment:
Why do we need this setting? We should be able to determine the path just by using `getBinPath` like you have done in the `goDebugConfiguration.ts` file

Reply:
If we are already passing it via the debug configuration. We might as well add it to the schema. Otherwise it's still possible to specify it but VS Code will warn about it.

We are determining the path via `getBinPath`, this just allows to override it.